ext/build.sh

```diff
@@ -55,4 +55,14 @@ cd NemaTode
 cmake .
 make -j${numCPU}
 make install
-popd
\ No newline at end of file
+popd
+
+# Nats C API
+pushd /tmp
+git clone https://github.com/nats-io/nats.c
+cd nats.c
+cmake . -DNATS_BUILD_NO_SPIN=ON
+make -j${numCPU}
+make install
```

Comment:
After we install this dependency, we should delete the source code.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/565/commits/a3b68aa8789a5a9d5e17bd3807628f4f76026d08